src/components/SettingsMenu.tsx

```diff
@@ -1,17 +1,11 @@
-import {
-  IconButton,
-  Menu,
-  MenuButton,
-  MenuList,
-  ThemeTypings,
-} from "@chakra-ui/react";
+import { IconButton, Menu, MenuButton, MenuList } from "@chakra-ui/react";
 import { RiSettings2Line } from "react-icons/ri";
 import { useIntl } from "react-intl";
 import LanguageMenuItem from "./LanguageMenuItem";
 import { useRef } from "react";
 
 interface SettingsMenuProps {
-  variant?: ThemeTypings["components"]["Menu"]["variants"];
+  variant?: string;
```

Comment:
Did the theme type cause issues?

Reply:
Yes!
<img width="722" alt="Screenshot 2024-07-04 at 13 13 17" src="https://github.com/microbit-foundation/ml-trainer/assets/145345672/a89dd2e4-0c07-4966-af54-b2a6b84b8de9">

Reply:
I have updated it here: https://github.com/microbit-foundation/ml-trainer/pull/267/commits/8c7953e38673c3599bff1ec60a26fd55e921fe2c
Strange that I don't get that typescript error in the Python Editor 🤔

Reply:
Fixed in eda9b836d612af5e1440801d1cce59510145410d